FridgeExercise/Program.cs

```diff
@@ -1,2 +1,242 @@
+				Console.WriteLine("There is no such a kosherType: " + KosherTypeToEat);
+
+		}
+		public void InsertItem(Refrigerator fridge)
+		{
+			int day = 0, month = 0, year = 0;
+			Console.WriteLine("Insert an Item:");
+			Console.WriteLine("Name:");
+			string name = Console.ReadLine();
+			Console.WriteLine("Expired date:in this tamplate:dd-mm-yyyy");
+			string expireDate = Console.ReadLine();
+			string[] dateSeperated = expireDate.Split('-');
+			if (dateSeperated.Count() == 3)
+			{
+				try
+				{
+					var checkDate = new DateTime(Int32.Parse(dateSeperated[2]), Int32.Parse(dateSeperated[1]), Int32.Parse(dateSeperated[0]));
+					day = Int32.Parse(dateSeperated[0]);
+					month = Int32.Parse(dateSeperated[1]);
+					year = Int32.Parse(dateSeperated[2]);
+				}
+				catch
+				{
+					Console.WriteLine("The date is not correct");
+					return;
+				}
+
+			}
+			Console.WriteLine("How much room it takes in square meters ?:enter just a double number");
+			double spaceItTakes = double.Parse(Console.ReadLine());
+			if (spaceItTakes > 0)
+			{
+				Console.WriteLine("Is it Dairy Parve Or Meat:notice that you spell it right");
+				string kosherType = Console.ReadLine();
+				Console.WriteLine("Is it a Drink or Food:notice that you spell it right");
+				string type = Console.ReadLine();
+				if (type == "Food" || type == "Drink")
+				{
+					if (kosherType == "Parve" || kosherType == "Dairy" || kosherType == "Meat")
+					{
+						Item itemToInsert = new Item(name, year, month, day, spaceItTakes, kosherType, type);
+						fridge.PutItemInTheFridge(itemToInsert, fridge.Shelves);
+					}
+					else
+						Console.WriteLine("There is no such a type:" + type);
+				}
+				else
+					Console.WriteLine("There is no such a kosherType: " + kosherType);
+
+			}
+
+		}
+
+		static void Main(string[] args)
+		{
+			var ThisProgramObject = new Program();
+			List<Refrigerator> allRefrigerators = new List<Refrigerator>();
+
+			Refrigerator refrigerator2 = new Refrigerator("Sharp", "write", 6);
+			allRefrigerators.Add(refrigerator2);
+			refrigerator2.Shelves.Add(new Shelf(1, 30));
+			refrigerator2.Shelves.Add(new Shelf(2, 30));
+			refrigerator2.Shelves.Add(new Shelf(3, 30));
+			refrigerator2.Shelves.Add(new Shelf(4, 25));
+			refrigerator2.Shelves.Add(new Shelf(3, 10));
+			refrigerator2.Shelves.Add(new Shelf(4, 17.5));
+			Item item21 = new Item("milk", 2023, 10, 15, 20, "Dairy", "Drink");
+			Item item22 = new Item("chily", 2023, 10, 22, 10, "Parve", "Drink");
+			Item item23 = new Item("meat", 2023, 10, 17, 30, "Meat", "Food");
+			Item item24 = new Item("chocolate", 2023, 10, 20, 5, "Dairy", "Drink");
+			refrigerator2.PutItemInTheFridge(item21, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item22, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item23, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item24, refrigerator2.Shelves);
+
+
+
+			Refrigerator refrigerator=new Refrigerator("samsung","gray",4);
```

Comment:
```suggestion
			Refrigerator refrigerator = new Refrigerator("samsung", "gray", 4);
```
Please make sure to use spaces in the appropriate places - before and after the "=", and after ","
In addition, I would probably create an enum for the RefrigeratorType and perhaps the RefrigeratorColor as well.